testsuite/cluster-test/src/cluster.rs

```diff
@@ -157,6 +169,63 @@ impl Cluster {
         &self.mint_key_pair
     }
 
+    fn account_key(&self) -> AccountKey {
```

Comment:
I assume this part is moved out from old cluster test tx_emitter?

Reply:
yep